code/_onclick/ai.dm

```diff
@@ -125,5 +125,14 @@
 	AltClick(user)
 	return
 
+/client/MouseEntered(object,location,control,params)
```

Comment:
This really isn't acceptable. This would mean that nearly every time anyone on the server moved their mouse, their client would have to alert the server. You need to find another method.

Reply:
stealth log-gunking

Reply:
Wouldn't mess with the logs. Would just increase traffic, by a lot. But anyway there's a relatively simple way to make this change without defining this proc which I explained to the author elsewhere